.github/workflows/main.yml

```diff
@@ -60,15 +57,9 @@ jobs:
       matrix:
         ruby:
           - 2.7
-          - 2.6
-          - 2.5
-          - jruby
+          - 3.0
```

Comment:
I'd add all the other versions here as well (2.8, 2.9).

Reply:
There is no 2.8 or 2.9, latest version of Ruby 2 is 2.7.6.

Reply:
Oh boy, you're right, my mind was somewhere else. Ignore this 😅